### DISCUSSION TITLE
Oracle driver for smartagent/sql receiver

### DISCUSSION THREAD

**Discussion by zchef2k** [signalfx/splunk-otel-collector#2341] [Q&A] [answered]
Would like to see an additional 'oracle' driver to those listed [here ](https://docs.splunk.com/Observability/gdi/sql/sql.html#nav-SQL)for the smartagent/sql receiver to metricize query output. I don't see this requested previously and would satisfy a lot of use cases.

If this is in the wrong place, please let me know and I'll resubmit.

Thanks everyone.

**Comment by atoulme** [comment, marked as answer]
You can use this receiver to query, it has the oracle driver.
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/sqlqueryreceiver

**Reply by zchef2k** [reply]
I didn't think to look in contrib. Working well for me now, thank you.
